website/content/docs/secrets/kmip-profiles.mdx

```diff
@@ -171,8 +171,72 @@ Vault implements version 1.4 of the following Key Management Interoperability Pr
+    | Symmetric Key [KMIP-SPEC 2.2.2][kmip-spec-2.2.2]                       | ✅        |
+    | Key Format Type [KMIP-SPEC 9.1.3.2.3][kmip-spec-9.1.3.2.3]             | ✅        |
+
+  3. Supports the following objects:
+
+    | Object                                                              | Supported | Notes |
+    | --------------------------------------------------------------------| :-------: | :---: |
+    | Public Key [KMIP-SPEC 2.2.3][kmip-spec-2.2.3]                       | ✅        |  Vault 1.13 |
+    | Private Key [KMIP-SPEC 2.2.4][kmip-spec-2.2.4]                      | ✅        |  Vault 1.13 |
+    | Process Start Date [KMIP-SPEC 3.25][kmip-spec-3.25]                 | ✅        |  Vault 1.11 |
+    | Key Format Type [KMIP-SPEC 9.1.3.2.3][kmip-spec-9.1.3.2.3]          | ✅        |        |
+
+  4. Supports the following attributes:
+
+    | Attribute                                                              | Supported | Notes |
+    | -----------------------------------------------------------------------| :-------: | :---: |
+    | Cryptographic Algorithm [KMIP-SPEC 3.4][kmip-spec-3.4]                 | ✅        |       |
+    | Object Type [KMIP-SPEC 3.3][kmip-spec-3.3]                             | ✅        |       |
+    | Process Start Date [KMIP-SPEC 3.25][kmip-spec-3.25]                    | ✅        | Vault 1.11 |
+    | Protect Stop Date [KMIP-SPEC 3.26][kmip-spec-3.26]                     | ✅        | Vault 1.11 |
+
+  5. Supports the following message encoding:
+
+    | Message Encoding                                                                     | Supported | Notes |
+    | -------------------------------------------------------------------------------------| :--------:|:-----:|
+    | Cryptographic Algorithm [KMIP-SPEC 9.1.3.2.13][kmip-spec-9.1.3.2.13] with values:    |           |       |
+    | i. RSA                                                                               | ✅        | Vault 1.13 |
+    | Object Type [KMIP-SPEC 9.1.3.2.12][kmip-spec-9.1.3.2.12] with value:                 |           |        |
+    | i. Public	 Key                                                                       | ✅        | Vault 1.13 |
+    | ii. Private	 Key                                                                     | ✅        | Vault 1.13 |
+    | Key Format Type [KMIP-SPEC 9.1.3.2.3][kmip-spec-9.1.3.2.3] with value:               |           |        |
+    | i. PKCS#1                                                                            | ✅        | Vault 1.13 <br/> Supported for Private and Public Keys|
+    | ii. PKCS#8                                                                           | ✅        | Vault 1.13 <br/> Supported for Private Key|
+    | iii. Transparent RSA Public Key                                                      | ✅        | Vault 1.13 |
+    | iv. Transparent RSA Private Key                                                      | ✅        | Vault 1.13 |
+
```

Comment:
I think we should include a note here stating that key format type X509 is also supported for public keys.

Reply:
This table mimics the server conformant requirements as is from the spec documentation, and X509 is not in the table (http://docs.oasis-open.org/kmip/profiles/v1.4/os/kmip-profiles-v1.4-os.html#_Toc491431516)
Will add a note under Asymmetric Key Lifecycle Server section in kmip.mdx page, does that sound good?

Reply:
Wouldn't it make sense for the note to be here? Maybe after "7. MAY support..."

Reply:
Ended up adding it under supported key format types. It made more sense to add there :)